HomeWork10/HomeWork10/Repositories/Abstructs/IHouseholdRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using HomeWork10.Entities;
+using HomeWork10.Models.Enums;
+using System.Drawing;
+
+namespace HomeWork10.Repositories.Abstructs
+{
+    internal interface IHouseholdRepository
+    {
+        public string AddHousehold(string name, int consumes, int power, BrendEntity brend, Color color, TypeHousehold typeAppliance);
```

Comment:
If you have more than 5 input parameter, you need to consider adding new Model/Entity for this request

Reply:
it`s verry good idea!!!